app/models/group.rb

```diff
@@ -24,6 +24,16 @@ class Group < ActiveRecord::Base
   validates :letter, presence: true
   validates_uniqueness_of :letter, scope: :conference_id
 
+  def self.to_csv
+    attributes = Group.attribute_names
```

Comment:
We don't actually wan't to export a group, since that will just give us one row.
What we really want to do is export the students that belong to a group (I'll direct message you)!